### ISSUE TITLE
App is not showing ads

### ISSUE THREAD

**Issue by yash3057** [akgupta0777/Kivy-Admob-Ads-Tutorial#2]
I compiled your app from your github using your buildozer.spec file but that app is not showing ads please help me to solve this problem

**Comment by akgupta0777** [comment]
You can use the apk provided in the builds folder of the github repository.It will show ads.

**Comment by akgupta0777** [comment]
The problem is that the kivy code written for the simple app is very old and not maintained as it is for educational purposes.
Thats why I have uploaded the apk which supports for all android versions and doesn't depend on code because that is already compiled.

**Comment by yash3057** [comment]
How can I put ads in my apk then please provide any suggestions or solutions

**Comment by akgupta0777** [comment]
the code is same as it is. The only thing that changes is the buildozer.spec file.
I will update it and you can get the ads if you use the same spec file.

**Comment by akgupta0777** [comment]
I am getting a lot of queries for ads not working on android 11 devices and some Samsung devices running android 10 or above. I will update you tomorrow with a fix to this issue or a solution.

**Comment by yash3057** [comment]
I am trying to use another module name kivads it says that it works with latest Android version if it doesn't work than I will use your buildozer.spec file

**Comment by yash3057** [comment]
I will inform you if this module works

**Comment by akgupta0777** [comment]
Sure try that module if it fits your needs.

**Comment by akgupta0777** [comment]
kivads is actually the same as kivmob by the way

**Comment by yash3057** [comment]
Yes it's same and it is working ads are shown on my device now

**Comment by akgupta0777** [comment]
great happy to know that ads are working. But I am investigating the cause why it is not working on KivMob. Even I am stuck at gradle build failed error on compiling this app to apk using kivmob.

**Comment by yash3057** [comment]
Your file was compiling on my pc but when using that app on Android it would not show ads . In logcat it showed that all function were called but ads were no t showing

**Comment by akgupta0777** [comment]
Are you talking about the latest file changes that made yesterday. 
Google changed meta data and gradle dependencies for ads so now you have to build the app with the latest changes I did in buildozer.spec see it

**Comment by akgupta0777** [comment]
But when I build my simple app again I don't know why its failing to package

**Comment by yash3057** [comment]
Comment out this line in your code and then try again
'self.theme_cls.theme_style="Dark" '
And your buildozer.spec file is same as mine

**Comment by akgupta0777** [comment]
No need I fixed this bug checkout code of main.py. I don't want to lose Dark Theme 😅😁

**Comment by yash3057** [comment]
Ok!!

**Comment by yash3057** [comment]
I got this problem again this time the code in your repository not compiling to .apk earlier it it was compiling to .apk but ads were not shown this time apk itself is not compiling buildozer is giving error

**Comment by akgupta0777** [comment]
Yes that is some issue with the gradle with buildozer. Don't know what is happening to the app either.

**Comment by yash3057** [comment]
What could I do to solve this issue the same issue is happening with kivads

**Comment by akgupta0777** [comment]
Sorry my friend I don't know even. I tried 10 different ways to solve this issue but got no solution. At the end I got frustration.

**Comment by yash3057** [comment]
I solve the issue . I think you should also try this ! You have to clone python for android from here "https://github.com/kivy/python-for-android"
And then from the file :
"python-for-android/pythonforandroid/bootstraps/common/build/gradle/wrapper/gradle-wrapper.properties"
Change the line :
"distributionUrl=https\://services.gradle.org/distributions/gradle-6.4.1-all.zip"
To
"distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip"
 This solution is taken from kivads documentation

**Comment by yash3057** [comment]
And yeah one more step in your buildozer.spec file in the
 "p4a.sourcedir="
You have to give the location of your this modified 
"python-for-android"

**Comment by akgupta0777** [comment]
@yash3057 did you managed to build apk of my app.
If everything worked submit a Pull request with whatever changes you have done so that this repo can help others too.

**Comment by yash3057** [comment]
Ok I will compile and submit

**Comment by akgupta0777** [comment]
Thanks @yash3057
